content/docs/tasks/security/auth-sds/index.md

```diff
@@ -80,3 +82,11 @@ $ kubectl delete ns foo
 $ kubectl delete ns bar
 $ kubectl delete -f istio-auth-sds.yaml
 {{< /text >}}
+
+## Caveats
+
+Currently, the SDS identity provision flow has the following caveats:
+
+* The control plane security has to be turned off. Enabling SDS with control plane security turned on is working in progress.
+
+* The support clusters smoothly migrating from using the secret volume mount to using the SDS is working in progress.
```

Comment:
What's the caveat here? I'm guessing that the migration is not smooth but the text does not clearly convey that idea. 
```suggestion
* There is no smooth migration for support clusters to stop using the secret volume mount approach. A smooth migration for the support clusters to use the SDS approach is a work in progress.
```